code/src/WijmoProvider/Columns/AbstractProviderColumnEditor.ts

```diff
@@ -70,6 +74,12 @@ namespace WijmoProvider.Column {
             //Save the editor on config =D
             this.config.editor = this._editor;
 
+            if (this.hasConditionalFormat) {
+                this._setConditionalFormat(
+                    this.editorConfig['conditionalFormat']
```

Comment:
This string, should then be a constant. Second time that a string is in the code :)

Reply:
Fixed it